quinn/Cargo.toml

```
@@ -53,11 +53,11 @@ unwrap = "1.2.1"
 url = "2"
 
 [[example]]
-name = "server"
+name = "https_server"
```

nit: I feel like the `s` is unnecessary, since no unencrypted form exists.
```suggestion
name = "http_server"
```